apps/extension/src/features/accounts/account-card.tsx

```diff
@@ -0,0 +1,240 @@
+import type { RouterOutputs } from "@agreeto/api";
+import { EventColorUserRadix } from "@agreeto/api/types";
+import {
+  Button,
+  Dialog,
+  DropdownMenu,
+  GoogleLogo,
+  MicrosoftLogo,
+} from "@agreeto/ui";
+import * as RadioGroup from "@radix-ui/react-radio-group";
+import * as Tooltip from "@radix-ui/react-tooltip";
+import type { FC } from "react";
+import { AiOutlineMore } from "react-icons/ai";
+import { HiCheckCircle } from "react-icons/hi";
+import resolveConfig from "tailwindcss/resolveConfig";
+
+import { trpcApi } from "~features/trpc/api/hooks";
+
+import tailwindConfig from "./../../../tailwind.config";
+
+const fullConfig = resolveConfig({
+  ...tailwindConfig,
+  content: ["./src/**/*.{html,js,ts,tsx}"],
+});
+
+export const themeColors = fullConfig.theme?.colors as Record<
+  EventColorUserRadix & string,
+  Record<number, string>
+>;
+
+type Account = RouterOutputs["user"]["myAccounts"]["accounts"][number];
+
+const AccountCard: FC<{
+  account: Account;
+}> = ({ account }) => {
+  const initials = account.email?.substring(0, 2).toLocaleUpperCase();
+
+  const { data: user } = trpcApi.user.me.useQuery();
+
+  const utils = trpcApi.useContext();
+  const { mutateAsync: updateEventColor } =
+    trpcApi.account.updateColor.useMutation({
+      onSuccess() {
+        utils.account.me.invalidate();
+        utils.event.all.invalidate();
+        utils.user.myAccounts.invalidate();
+        utils.user.me.invalidate();
+      },
+    });
+
+  return (
+    <Tooltip.Provider>
+      <div className="flex justify-between py-6 pr-6 space-x-4 border rounded-lg h-30 border-mauve-6 pl-9">
+        <div className="flex w-full gap-4">
+          {/* Avatar w/ initials */}
+          <div
+            className="self-center w-10 h-10 leading-10 text-center rounded-full"
+            style={{
+              backgroundColor: themeColors[account.eventColor][7],
+              color: themeColors[account.eventColor][11],
+            }}
+          >
+            {initials}
+          </div>
+          {/* Email, Colors & Badges */}
+          <div className="flex flex-col items-start flex-grow px-10 ">
+            {/* this div makes sure that the children have the same width as determined by colors radio group */}
+            <div>
+              {/* Email and organizer badge */}
+              <div className="flex justify-between">
+                <div className="text-sm font-normal leading-5 flex items-center gap-2">
+                  {account.provider === "google" ? (
+                    <GoogleLogo className="h-4 w-4" />
+                  ) : (
+                    <MicrosoftLogo className="h-4 w-4" />
+                  )}
```

Comment:
I added a little logo here, what'ya think? 
![CleanShot 2022-11-29 at 01 58 19](https://user-images.githubusercontent.com/51714798/204412297-e659f3c7-242e-4003-90bb-524b20c38bc8.png)

Reply:
I like it! 🙏